coa-helm/values.yaml

```diff
@@ -70,3 +70,10 @@ identity:
   name: RUNTIME_PARAMETER
   resourceId: RUNTIME_PARAMETER
   clientId: RUNTIME_PARAMETER
+
+workloadIdentity:
+  enabled: false
+  serviceAccountName: RUNTIME_PARAMETER
+
+postgres:
+  podLocalDatabaseEnabled: true

```

Comment:
This is a repetition of the value in the library chart but I think it's convenient to include here for readability and potentially setting a new default for this chart.